cmd/fireacme/cli/main.go

```diff
@@ -81,7 +81,7 @@ func Main() {
 	derr.Check("acme-blockchain", RootCmd.Execute())
 }
 
-var startCmdExample = `fireacme start extractor`
+var startCmdExample = `fireacme start reader`
```

Comment:
Pretty sure this should be
```suggestion
var startCmdExample = `fireacme start reader-node`
```

Test it to see.